examples/text_to_image_LCM.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+
+from diffusers import DiffusionPipeline
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Simple demo of LCM image generation.")
+    parser.add_argument(
+        "--prompt",
+        type=str,
+        default="Self-portrait oil painting, a beautiful cyborg with golden hair, 8k",
+    )
+    parser.add_argument(
+        "--model_id",
+        type=str,
+        default="SimianLuo/LCM_Dreamshaper_v7",
+    )
+    parser.add_argument("--height", type=int, default=768)
+    parser.add_argument("--width", type=int, default=768)
+    parser.add_argument("--steps", type=int, default=4)
+    parser.add_argument("--warmup", type=int, default=1)
+    parser.add_argument("--seed", type=int, default=1)
+    parser.add_argument(
+        "--disable", action="store_true", help="Disable Onediff speeding up"
+    )
+    args = parser.parse_args()
+    return args
+
+
+args = parse_args()
+
+if not args.disable:
+    from onediff.infer_compiler import oneflow_compile
+import torch
+
+pipe = DiffusionPipeline.from_pretrained(args.model_id)
+
+pipe.to(torch_device="cuda", torch_dtype=torch.float16)
+if not args.disable:
+    pipe.unet = oneflow_compile(pipe.unet)
```

Comment:
RTX 3090, fp16, steps=4：
- origin:  28.5 it/s
- with oneflow_compile in onediff: 14.8 it/s